python-threatexchange/threatexchange/content_type/photo.py

```diff
@@ -4,9 +4,11 @@
 """
 Wrapper around the video content type.
 """
+from PIL import Image
+import io
 
 from .content_base import ContentType
-
+from threatexchange.content_type.content_base import RotationType
```

Comment:
nit: We shouldn't be using relative imports here, and they are both from the same file.